lib/roby/plan.rb

```diff
@@ -1609,6 +1609,17 @@ def finalize_event(event, timestamp = nil)
             event.finalized!(timestamp)
         end
 
+        # Copy plan marks from one into the other, additive only i.e. it does not remove
+        # marks from the destination task
```

Comment:
Summary: "Apply to `to` the marks (permanent, mission) of `from`"

If you really want to mention that it does not remove the marks of `from`, then say it in the detailed documentation